Lombiq.TrainingDemo/Drivers/DemoSettingsDisplayDriver.cs

```diff
@@ -32,10 +32,10 @@ public DemoSettingsDisplayDriver(IAuthorizationService authorizationService, IHt
 
     // Here's the EditAsync override to display editor for our site settings on the Dashboard. Note that it has a sync
     // version too.
-    public override async Task<IDisplayResult> EditAsync(DemoSettings section, BuildEditorContext context)
+    public override async Task<IDisplayResult> EditAsync(ISite model, DemoSettings section, BuildEditorContext context)
```

Comment:
You can use `SiteDisplayDriver<TSettings>` as the base for all of these `ISite`-using `SectionDisplayDriver`s.